admin-dev/themes/new-theme/js/stock-page/components/header/search-filter.vue

```diff
@@ -0,0 +1,8 @@
+<template>
+  <select class="form-control m-b-3 search-filter">
+    <option selected>Open this select menu</option>
```

Comment:
@aleeks We will do that in the next step, probably with a json generated on server side. ping @Shudrum @thierrymarianne